containers/pre_built_workers/node/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM node:10-buster
+
+RUN mkdir -p /pre
+WORKDIR /pre
+
+ARG REPOSITORY=grpc/grpc-node
```

Comment:
As we have talked about since it is a ARG, it takes a bit work to access after the build time. We might want to save them somewhere for future access.  I had something like 
```
RUN echo 'COMMIT SHA' > GRPC_GIT_COMMIT.txt
RUN git rev-parse $GITREF >> GRPC_GIT_COMMIT.txt
```
in pr #113. You could do something else too. :) Also it might be helpful to read the comments from Jan for pr#113. :)

Reply:
Done! thanks for reference, those bazel-related comments help me improving the prebuilt image for python!